drizzlepac/hlautils/astrometric_utils.py

```diff
@@ -767,36 +708,26 @@ def compute_photometry(catalog, photmode):
     return catalog
 
 
-def filter_catalog(catalog, **kwargs):
-    """ Create a new catalog selected from input based on photometry
+def filter_catalog(catalog, bright_limit=1.0, max_bright=None, min_bright=20, colname="vegamag"):
+    """ Create a new catalog selected from input based on photometry.
 
     Parameters
     ----------
```

Comment:
Missing description of `catalog` parameter.

Reply:
Added...